samples/AWS/AWS.AppHost/Directory.Build.props

```
@@ -0,0 +1,8 @@
+<Project>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
+  <!-- NOTE: This line is only required because we are using P2P references, not NuGet. It will not exist in real apps. -->
+  <Import Project="../../../src/Aspire.Hosting/build/Aspire.Hosting.props" />
```

Please use `$(RepoRoot)` variable instead
```suggestion
  <Import Project="$(RepoRoot)/src/Aspire.Hosting/build/Aspire.Hosting.props" />
```

Ditto other places.

---

Done